README.md

```diff
@@ -57,7 +61,7 @@ You can propose popular transformer rules to be included in the plugin by [sugge
 
 **How do I post articles to Instant Articles after plugin is installed?**
 
-You can re-publish existing articles (simply edit + save) or post new articles in order to submit them to Instant Articles. After you have 10 articles added, you will be able to submit them for review.
+You can re-publish existing articles (simply edit + save) or post new articles in order to make them available as Instant Articles. After you have 5 articles added and shared on Facebook, you will be able to submit them for review.
```

Comment:
The first sentence is still inaccurate. Editing + saving the article does nothing anymore. You need to share it (even without editing+saving). Check my previous comment. Also, we now require 10 articles, not 5.